src/codeu/chat/common/ServerInfo.java

```diff
@@ -0,0 +1,29 @@
+package codeu.chat.common;
+import codeu.chat.util.Uuid;
+import java.io.IOException;
+
+public final class ServerInfo {
+
+  private final static String SERVER_VERSION = "1.0.0";
+
+  public final Uuid version;
+
+  //sets the server version to either null or
+  public ServerInfo() {
+    Uuid temp = null;
+    try{
+      temp =  Uuid.parse(SERVER_VERSION);
+    } catch(IOException e){
+      System.out.println("Server could not parse the user id");
```

Comment:
My apologies, I should have clarified. I'm commenting specifically on the string you're printing in line 17, `"Server could not parse the user id"`. Are we parsing a user id?